USAGE.md

```diff
@@ -364,6 +364,7 @@ print_r($page->getError());
 
 For MediaWiki API errors, the array contains the 'code' and 'info' key/value pairs [defined by the API](https://www.mediawiki.org/wiki/Special:MyLanguage/API:Errors_and_warnings#Errors).  For other errors, the following key/value pairs are returned:
 * 'login' for Wikimate authentication problems
+* 'auth' for Wikimate authentication problems
```

Comment:
Perhaps we could add an HTML comment (`<!-- -->`) in the line above to explain the temporary duplication.

Reply:
(A "TODO: remove when..." note would work, btw.)

Reply:
Good idea 2, done.